gateway/webhook/webhookTransformer.go

```diff
@@ -21,22 +25,25 @@ import (
 )
 
 type sourceTransformAdapter interface {
-	getTransformerEvent(authCtx *gwtypes.AuthRequestContext, body []byte) ([]byte, error)
+	getTransformerEvent(authCtx *gwtypes.AuthRequestContext, eventRequest []byte) ([]byte, error)
 	getTransformerURL(sourceType string) (string, error)
+	getAdapterVersion() string
 }
 
+// ----- v1 adapter ---------
+
 type v1Adapter struct{}
 
 type V1TransformerEvent struct {
-	Event  json.RawMessage       `json:"event"`
-	Source backendconfig.SourceT `json:"source"`
+	EventRequest json.RawMessage       `json:"event"`
+	Source       backendconfig.SourceT `json:"source"`
 }
 
-func (v1 *v1Adapter) getTransformerEvent(authCtx *gwtypes.AuthRequestContext, body []byte) ([]byte, error) {
+func (v1 *v1Adapter) getTransformerEvent(authCtx *gwtypes.AuthRequestContext, eventRequest []byte) ([]byte, error) {
 	source := authCtx.Source
 
 	v1TransformerEvent := V1TransformerEvent{
-		Event: body,
+		EventRequest: eventRequest,
```

Comment:
this renaming i.e. we need to change a lot of integrations or we need to change in the controller/service layer.

Reply:
We won't have to change anything on the transformer as the json that is being build from the structure is till the same
`EventRequest json.RawMessage `json:"event"`` -> here the json field is still **event**